README.md

```diff
@@ -1,14 +1,61 @@
-# XAITK - Saliency
+![xaitk-logo](./docs/figures/xaitk-wordmark.png)
```

Comment:
It might make sense to still have a text header that includes the word "saliency" in some form or another, since we might eventually standardize the headers of all repos under the XAITK github namespace (e.g. autonomy, web-api, web-demo, etc.) to have this same logo image.